saleor/graphql/schema.graphql

```diff
@@ -2557,6 +2557,8 @@ type Mutation {
   shippingPriceBulkDelete(ids: [ID]!): ShippingPriceBulkDelete
   shippingPriceUpdate(id: ID!, input: ShippingPriceInput!): ShippingPriceUpdate
   shippingPriceTranslate(id: ID!, input: NameTranslationInput!, languageCode: LanguageCodeEnum!): ShippingPriceTranslate
+  shippingPriceExcludeProducts(id: ID!, input: ShippingPriceExcludeProductsInput!): ShippingPriceExcludeProducts
+  shippingPriceRemoveProductFromExclude(id: ID!, input: ShippingPriceRemoveProductFromExcludeInput!): ShippingPriceRemoveProductFromExclude
```

Comment:
Maybe instead of the `input` object, we could have just the `products` list here? This input doesn't have any other fields.